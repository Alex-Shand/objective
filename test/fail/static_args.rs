#[objective::objective]
trait Test {
    fn test(a: A, b: B, c: C) -> D {
    }
}

fn main() {}
