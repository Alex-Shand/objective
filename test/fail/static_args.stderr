error: Static trait methods aren't object safe
 --> test/fail/static_args.rs:3:5
  |
3 |     fn test(a: A, b: B, c: C) -> D {
  |     ^^
